### ISSUE TITLE
SILVA inputs problem with Escherichia and Shigella

### ISSUE THREAD

**Issue by jjoropezav** [micom-dev/databases#1]
Hi again @cdiener thank you always for your help, This is not an issue from micom but from the silva and the databases properly. 
I ran my simulations with micom and found surprised that the genus escherichia neither shigella grew in my models using AGORA or AGORA2.

Found that the problem is that my qiime artifacts were trained with SILVA, currently SILVA have a new genus established as Escherichia-Shigella. 

https://www.biorxiv.org/node/2170154.full
https://www.arb-silva.de/browser/ssu-138/AM184304

i modified the manifest file to correct this, if you want i can send them to you.

Thanks a lot.

**Comment by cdiener** [comment]
Yeah, good point. All the databases are built on top of a specific taxonomic naming scheme, and you will get the best results if you match your data to it. Default AGORA uses refseq, so you need the NCBI taxonomy. We built a genus-level classifier [based on that for a course](https://github.com/Gibbons-Lab/isb_course_2022/blob/main/ncbi-refseq-genus-515f-806r.qza). Or you could use the GTDB classifier and use the database versions using that scheme.
